net/shorewall6-lite/Makefile

```diff
@@ -59,19 +58,19 @@ define Build/Compile
 endef
 
 define Package/shorewall6-lite/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_DIR) $(1)/etc/shorewall6-lite/state
-	$(INSTALL_DIR) $(1)/usr/share
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface/
```

Comment:
Again, please don't gratuitously change the order of lines.  Go for minimal change sets.

Reply:
This was a one time change as suggested by a reviewer.